userdocs/requirements.txt

```
@@ -1,4 +1,4 @@
 mkdocs                  == 1.1
-mkdocs-material         == 5.1.3
+mkdocs-material         == 5.3.0
```

```suggestion
mkdocs-material         == 5.3.3
```

There apparently were some search fixes since 5.3.0: https://github.com/squidfunk/mkdocs-material/releases